runs/auth/run.sh

```diff
@@ -0,0 +1,2 @@
+docker rm -f auth
```

Comment:
This because it is not a `tty` and ibazel does not allow bash traps.

If you want to know more reach out 😂omg

cc. https://github.com/bazelbuild/bazel-watcher/issues/291